query-connector/src/app/ui/designSystem/TabGroup/tabGroup.tsx

```diff
@@ -2,13 +2,14 @@
 import React, { useState } from "react";
 import classNames from "classnames";
 import styles from "./tabGroup.module.scss";
+import { User, UserGroup } from "@/app/models/entities/users";
 
 export type Tab = {
   access?: string[];
   label: string;
   path?: string;
   onClick?: (e: React.MouseEvent<HTMLElement>) => void;
-  renderContent?: () => JSX.Element;
+  renderContent?: (users?: User[], userGroups?: UserGroup[]) => JSX.Element;
```

Comment:
is this method getting called somewhere? could we delete it if not?

if it is somehow, think we want to rethink the type here since I wouldn't expect tab UI only to render users / user groups.

Reply:
Yep, it gets called in the `usersTable` [line 242](https://github.com/CDCgov/dibbs-query-connector/blob/df8ea1bf2bb682ad8889b1601da0be43b51b2d66/query-connector/src/app/(pages)/userManagement/components/usersTable/usersTable.tsx#L242)

Good point re: typing; I'll create a follow-up ticket to rework this unless there are objections to merging it as is?